assignment/README.md

```diff
@@ -55,6 +98,13 @@ are hard to use and think about. Global variables are a strategy for
 coping with that difficulty by breaking data out of the asynchronous
 context.
 
+Probable difficulties:
+1.joininig datasets spatially
```

Comment:
Turf can probably do this if you're just looking to label points with values from vectors they're inside of: http://turfjs.org/docs/#tag